password_security/models/res_users.py

```diff
@@ -13,7 +14,7 @@
 
 def delta_now(**kwargs):
     dt = datetime.now() + timedelta(**kwargs)
-    return fields.Datetime.to_string(dt)
+    return fields.Datetime.to_datetime(dt)
```

Comment:
I think this is already a datetime, isn't it? Shouldn't this be then just `return dt`?

Reply:
It's my bad. Fixed.